risc0/zkvm/src/exec/env.rs

```diff
@@ -131,18 +155,56 @@ impl<'a> ExecutorEnvBuilder<'a> {
     }
 
     /// Set a session limit, specified in number of cycles.
+    /// # Example
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder};
```

Comment:
Why are these commented out? Seems useful to have in the example.

Reply:
You're right -- going off of TimZ's feedback, I'm going to un-comment a majority of the omitted code, including includes.